lib/src/observable.dart

```diff
@@ -9,7 +9,7 @@ import 'util.dart';
 // ignore_for_file: non_constant_identifier_names
 
 // dart callback signature
-typedef Any = void Function(Pointer<Void>, Pointer<Uint32>, int);
```

Comment:
The change to the 2nd param is [questionable](https://github.com/objectbox/objectbox-c/blob/master/include/objectbox.h#L1543-L1546).

[It's an array of Uint32s](https://github.com/objectbox/objectbox-c/blob/master/include/objectbox.h#L56).

Reply:
> The change to the 2nd param is [questionable](https://github.com/objectbox/objectbox-c/blob/master/include/objectbox.h#L1543-L1546).
> 
> [It's an array of Uint32s](https://github.com/objectbox/objectbox-c/blob/master/include/objectbox.h#L56).
> 
> The array should be unpacked, and the entityId it contains should be used to trigger the specific callback(s) indexed by storePtr and entityId. I might have been a bit sloppy.

@Buggaboo I didn't change it.

Reply:
Sure. I'm on it.